android/src/main/AndroidManifest.xml

```diff
@@ -83,16 +83,6 @@
             </intent-filter>
         </activity>
 
-
-        <receiver
```

Comment:
I removed this level of indirection. I didn't see why it was needed, and things seem to work without it.

Reply:
Aren't these needed for IPNReceiver to receive these intents from external components (eg Tasker)?

Reply:
Ohh, that's why we had IPNReceiver! Okay, let me add it back in ...